unity3d/Assets/Swrve/SwrveSDK/SwrveNativeIOS.cs

```diff
@@ -428,12 +431,21 @@ private void showNativeConversation(string conversation)
     private void startNativeLocation()
     {
         try {
```

Comment:
We don't need to do that on iOS, those functions are just dummied up and return null / false, they don't fail nor give an error message on failing.